ludwig/features/image_feature.py

```diff
@@ -90,6 +92,31 @@
 ###
 # Image specific augmentation operations
 ###
+@register_augmentation_op(name="auto_augmentation", features=IMAGE)
+class AutoAugment(torch.nn.Module):
+    def __init__(self, config: AutoAugmentationConfig):
+        super().__init__()
+        self.auto_augmentation_method = config.method
+        self.augmentation_method = self.get_augmentation_method()
+
+    def get_augmentation_method(self):
```

Comment:
@ethanreidel If not too much trouble, just from the coding style perspective, `return` statements are terminal, so we would not need `elif`.  Would it be possible for you to make that change?  Thanks a lot!